app/controllers/feedbacks_controller.rb

```diff
@@ -32,4 +32,13 @@ def load_support_languages
     @support_languages = Language.where(support_available: true).order(:name)
   end
 
+  private
+
+  def feedback_params
+    params.require(:feedback).permit(
+      :comment, :email, :summary, :user_agent, :category, :approved,
```

Comment:
I can't see category or approved in the form and therefore should they be let in ?